### DISCUSSION TITLE
How to convert UniTask/UniTaskVoid to Task?

### DISCUSSION THREAD

**Discussion by LukasDavidSchwarz** [Cysharp/UniTask#511] [Q&A]
### Question:
I want to do the same thing as asked here #276: Await an async operation synchronously. In the issue, it is mentioned that I should convert UniTask to Task. How is that done? Does it also work for UniTaskVoid?

### Why?
I essentially want to do this in an `void GameShuttingDown()` method, to await the running task before quitting the game.

Edit: Possible duplicate of  #481

**Comment by gakkossphynx** [comment]
Anybody here ?
